indico/modules/rb_new/client/js/modules/landing/Landing.jsx

```diff
@@ -20,12 +20,16 @@ import {connect} from 'react-redux';
 import PropTypes from 'prop-types';
 import {Card, Checkbox, Form, Grid} from 'semantic-ui-react';
 import {Translate} from 'indico/react/i18n';
+import Carousel from 'indico/react/components/Carousel';
```

Comment:
export via indico/react/components/index.js?

even though i'm starting to feel like there wouldn't be any real disadvantage in not having that index.js and then importing it like you did

Reply:
I think that importing from `components` is cleaner, especially if you have to import several classes from the same module.